Assets/Scripts/Minigame/DragBlanket.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.Serialization;
+using System.Collections;
+
+namespace Minigame
+{
+    public class DragBlanket : MinigameBaseClass
+    {
+        [Header("Scene References")]
+        [SerializeField] private TextMeshProUGUI _tmpUGUI;

```

Comment:
Idea: since you are displaying messages for each of the minigames, it might be simpler to make a custom Prompt class that is addressable via static methods like "SetText(string text)", but we can discuss this later. this is NOT a priority, just a thought.